mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/store/JobArtifact.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.mantisrx.server.master.store;
+
+import io.mantisrx.shaded.com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.mantisrx.shaded.com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.Instant;
+import java.util.Map;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class JobArtifact {
+    @JsonProperty("artifactID")
+    ArtifactID artifactID;
+    @JsonProperty("project")
+    String project;
+    @JsonProperty("version")
+    String version;
+    @JsonProperty("createdAt")
+    Instant createdAt;
+    @JsonProperty("runtimeType")
+    String runtimeType;
+    @JsonProperty("dependencies")
+    Map<String, String> dependencies;
```

Comment:
I'm thinking depth 1 for now as `Map<dep_name, dep_version>` . What do you think?

Reply:
btw what's the usage for this? This might make the API response unexpectedly large.

Reply:
good point. The first use case for this is to include the mantis runtime lib + version without being too explicit about it (since other companies may build a uber jar and not having to specify the runtime)